cypress/integration/features/login_management/forgotPasswordDialog.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Visibility of dBildungscloud Password Reset dialog
```

Comment:
`forgotPassword.feature` is fine don't need to mention `Dialog` here

Reply:
if we remove 'Dialog' here then it may imply 'password recovery' functionality as well, but this feature is only about visibility of dialog box and its components.